test/test.jl

```diff
@@ -229,6 +230,20 @@ end
 println("Testing keyword arguments")
 testfunc13()
 
+function testfunc14(barglyphs)
+    n = 30
+    # full keyword argumetns
+    p = ProgressMeter.Progress(n, barglyphs=ProgressMeter.BarGlyphs(barglyphs))
```

Comment:
This is nice. It would be even nicer to add a test that uses this in conjunction with the other kwargs (I don't think it will make a difference now, but we should make sure that it doesn't in the future).

Also, a couple of tests that invoke both constructors of `BarGlyphs` would be nice, since they're part of the public interface.